src/common/messages/MessageApi.py

```diff
@@ -0,0 +1,93 @@
+"""Define public messaging API of the CANS application."""
+
+from json import JSONDecodeError, JSONDecoder, JSONEncoder
+from typing import Any
+
+from common.messages.CansMessage import CansMessage
+from common.messages.CansSerial import CansSerial
+from common.messages.exceptions.CansDeserializationError import (
+    CansDeserializationError,
+)
+from common.messages.exceptions.CansInternalError import CansInternalError
+
+
+async def cans_recv(socket: Any) -> CansMessage:
```

Comment:
Czy socket nie jest typu ws.WebSocketClientProtocol? 
Bo jeżeli chcemy abby interfejs przyjmował różnego typu sockety to można zrobić `Union[ws.WebSocketClientProtocol, ...]`